script.js

```diff
@@ -22,8 +22,8 @@ const displayData = () =>{
 
   return `
   <li>
-  <div class="item item--name">{task}</div>
-  <div class="item item--date">{due}</div>
+  <div class="item item--name">${task}</div>
+  <div class="item item--date">${due}</div>
```

Comment:
for template strings, you need the dollar sign in front of the brackets when injecting variables